examples/evaluation/huggingface-evaluation.ipynb

```diff
@@ -0,0 +1,1340 @@
+       "2  Mohammad Javad Zarif has spent more time with ...   \n",
+       "3  17 Americans were exposed to the Ebola virus w...   \n",
+       "4  Student is no longer on Duke University campus...   \n",
+       "5  College-bound basketball star asks girl with D...   \n",
+       "6  Amnesty's annual death penalty report catalogs...   \n",
+       "7  Andrew Getty's death appears to be from natura...   \n",
+       "8  Once a super typhoon, Maysak is now a tropical...   \n",
+       "9  Bob Barker returned to host \"The Price Is Righ...   \n",
+       "\n",
+       "                                         id  \\\n",
+       "0  f001ec5c4704938247d27a44948eebb37ae98d01   \n",
+       "1  230c522854991d053fe98a718b1defa077a8efef   \n",
+       "2  4495ba8f3a340d97a9df1476f8a35502bcce1f69   \n",
+       "3  a38e72fed88684ec8d60dd5856282e999dc8c0ca   \n",
+       "4  c27cf1b136cc270023de959e7ab24638021bc43f   \n",
+       "5  1b2cc634e2bfc6f2595260e7ed9b42f77ecbb0ce   \n",
+       "6  e2706dce6cf26bc61b082438188fdb6e130d9e40   \n",
+       "7  0d3c8c276d079c4c225f034c69aa024cdab7869d   \n",
+       "8  6222f33c2c79b80be437335eeb3f488509e92cf5   \n",
+       "9  2bd8ada1de6a7b02f59430cc82045eb8d29cf033   \n",
+       "\n",
+       "                                              inputs  \n",
+       "0  Please summarize the following article:\\n(CNN)...  \n",
+       "1  Please summarize the following article:\\n(CNN)...  \n",
+       "2  Please summarize the following article:\\n(CNN)...  \n",
+       "3  Please summarize the following article:\\n(CNN)...  \n",
+       "4  Please summarize the following article:\\n(CNN)...  \n",
+       "5  Please summarize the following article:\\n(CNN)...  \n",
+       "6  Please summarize the following article:\\n(CNN)...  \n",
+       "7  Please summarize the following article:\\n(CNN)...  \n",
+       "8  Please summarize the following article:\\n(CNN)...  \n",
+       "9  Please summarize the following article:\\n(CNN)...  "
+      ]
+     },
+     "execution_count": 51,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "eval_df = pd.DataFrame(dataset[\"test\"])\n",
+    "eval_df[\"inputs\"] = \"Please summarize the following article:\\n\" + eval_df[\"article\"]\n",
+    "\n",
+    "eval_df.head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "a40f09d2-91cf-4390-a606-786cf93e6a62",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Define Extra Metrics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create a custom LLM-judged metric named `answer_quality` using `make_genai_metric()`. We need to define a metric definition and grading rubric, as well as some examples for the LLM judge to use."
```

Comment:
Can some additional clarification be provided here that explains:
- What these metrics are
- Why they are important
- How the EvaluationMetric collection is constructed
- Where these instructional prompts come from (are these just for this tutorial, are they standard, how could a set of new criteria be developed, etc.)
- What the parameters are to `make_genai_metric` and how to use them

I think this section could use a great deal of detail which might end up being several sections and paragraphs of text (some of which might be worthwhile to hide under a 'details' drop down). Here's an example of how to create a dropdown:

``` html
<details>
    <div>
        <p>With our model, training arguments, and MLflow experiment set up, we are now ready to start the actual training process. This step involves initiating an MLflow run, which will encapsulate all the training activities and metrics.</p>
    </div>
    <div>
        <h4>Initiating the MLflow Run</h4>
        <ul>
            <li><strong>Starting an MLflow Run</strong>: We use <i>mlflow.start_run()</i> to begin a new MLflow run. This function creates a new run context, under which all the training operations and logging will occur.</li>
            <li><strong>Training the Model</strong>: Inside the MLflow run context, we call <i>trainer.train()</i> to start training our model. This function will run the training loop, processing the data in batches, updating model parameters, and evaluating the model.</li>
        </ul>
    </div>
    <div>
        <h4>Monitoring the Training Progress</h4>
        <p>During training, the <i>Trainer</i> object will output logs that provide valuable insights into the training progress:</p>
        <ul>
            <li><strong>Loss</strong>: Indicates the model's performance, with lower values signifying better performance.</li>
            <li><strong>Learning Rate</strong>: Shows the current learning rate used during training.</li>
            <li><strong>Epoch Progress</strong>: Displays the progress through the current epoch.</li>
        </ul>
    </div>
    <div>
        <p>These logs are crucial for monitoring the model's learning process and making any necessary adjustments. By tracking these metrics within an MLflow run, we can maintain a comprehensive record of the training process, enhancing reproducibility and analysis.</p>
    </div>
    <div>
        <p>In the next code block, we will start our MLflow run and begin training our model, closely observing the output to gauge the training progress.</p>
    </div>
</details>

```